### PR TITLE
[16.0][MIG] account_fiscal_position_partner_type: Migration to 16.0

### PR DESCRIPTION
**Pull request by RodrigoBM**
#325

### CODE REVIEW

**Comment by SoniaViciana** [comment]
LGFM!

**Comment by RodrigoBM** [comment]
@rafaelbn  can you add to issue https://github.com/OCA/account-fiscal-rule/issues/325 , thanks

File: account_fiscal_position_partner_type/tests/test_account_fiscal_position_partner_type.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields
+from odoo.tests import Form, common
+
+
+class TestAccountFiscalPositionPartnerType(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
```
Comment (ljsalvatierra-factorlibre) [inline account_fiscal_position_partner_type/tests/test_account_fiscal_position_partner_type.py]: ¿Could you use instead `setUpClass`?

**Review by ljsalvatierra-factorlibre — CHANGES_REQUESTED** [review]

File: account_fiscal_position_partner_type/tests/test_account_fiscal_position_partner_type.py (hunk 1 above)
Comment (RodrigoBM) [reply account_fiscal_position_partner_type/tests/test_account_fiscal_position_partner_type.py]: Done

**Review by ljsalvatierra-factorlibre — APPROVED** [review]

**Review by JavierIniesta — APPROVED** [review]

**Comment by RodrigoBM** [comment]
@rafaelbn can you merge it?, please

**Comment by RodrigoBM** [comment]
@pedrobaeza can you merge it?, please

**Review by rafaelbn — APPROVED** [review]
Thank you!

**Pull request merged**